src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1724,4 +1733,256 @@ void scaleInletFlows(EnergyPlusData &state, int const zoneNodeNum, int const spa
     spaceNode.MassFlowRateMin = zoneNode.MassFlowRateMin * frac;
     spaceNode.MassFlowRateMinAvail = zoneNode.MassFlowRateMinAvail * frac;
 }
+
+void ZoneEquipmentSplitterMixer::size(EnergyPlusData &state)
+{
+    bool anyAutoSize = false;
+    for (auto &thisSpace : this->spaces) {
+        if (thisSpace.fraction == DataSizing::AutoSize) {
+            anyAutoSize = true;
+            break;
+        }
+    }
```

Comment:
This whole loop:
```
bool anyAutoSize = std::any_of(spaces.begin(), spaces.end(), [](Space s){return s.fraction == AutoSize;});
```

Reply:
Ok, see if I did this right. Seems to work.